src/entitysdk/models/brain_region.py

```diff
@@ -5,10 +5,11 @@
 from pydantic import Field
 
 from entitysdk.models.core import Identifiable
+from entitysdk.models.taxonomy import Species, Strain
 from entitysdk.types import ID
 
 
-class BrainRegion(Identifiable):
+class NestedBrainRegion(Identifiable):
```

Comment:
Please don't change the types of the models because of the mapping between model <-> route. Can't we use BrainRegion everywhere in the client?